src/hooks/HooksRunner.js

```diff
@@ -45,7 +44,7 @@ function HooksRunner (projectRoot) {
  */
 HooksRunner.prototype.fire = function fire (hook, opts) {
     if (isHookDisabled(opts, hook)) {
-        return Q('hook ' + hook + ' is disabled.');
+        return Promise.resolve('hook ' + hook + ' is disabled.');
```

Comment:
This method resolves with a string, or throws an error (not rejects), or resolves with the result of running scripts, or rejects with error from running scripts? :grimacing: